### PR TITLE
Added right click refresh to folders & files

### PR DESCRIPTION
**Pull request by DivergentEuropeans**
Signed-off-by: Leanid Astrakou <lastrakou@rocketsoftware.com>

### CODE REVIEW

**Review by lchudinov — APPROVED** [review]
"Refresh" works well aside from an edge case when the folder, which I want to refresh, has been removed. I assume it's very easy to fix so I approve it now.

File: src/app/components/filebrowseruss/filebrowseruss.component.ts [hunk 1]
```diff
@@ -841,6 +848,46 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
...
+          node.mode = result.mode;
+          node.owner = result.owner;
+          node.group = result.group;
+          node.size = result.size;
+          node.ccsid = result.ccsid;
+          node.createdAt = result.createdAt;
+          return node;
+        },
+        e => {
+          if (e.status = 404) {
```
Comment (cleonty) [inline src/app/components/filebrowseruss/filebrowseruss.component.ts:875]: @DivergentEuropeans please fix a typo, use `==` or `===`

Comment (lchudinov) [inline src/app/components/filebrowseruss/filebrowseruss.component.ts:875]: @DivergentEuropeans please fix a typo, use `==`  or `===`

File: src/app/components/filebrowseruss/filebrowseruss.component.ts [hunk 2]
```diff
@@ -841,6 +848,46 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
...
+            this.snackBar.open("Failed to refresh '" + node.name + "' No longer exists or has been renamed.", 
+          'Dismiss', defaultSnackbarOptions);
+            this.removeChild(node);
+          } else if (e.status = 403) {
```
Comment (lchudinov) [inline src/app/components/filebrowseruss/filebrowseruss.component.ts:879]: @DivergentEuropeans please fix a typo, use `==`  or `===`

File: src/app/components/filebrowseruss/filebrowseruss.component.ts [hunk 3]
```diff
@@ -841,6 +848,46 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
...
+            this.snackBar.open("Failed to refresh '" + node.name + "' Permission denied.", 
+          'Dismiss', defaultSnackbarOptions);
+          } else if (e.status = 500) {
```
Comment (lchudinov) [inline src/app/components/filebrowseruss/filebrowseruss.component.ts:882]: @DivergentEuropeans please fix a typo, use `==`  or `===`

**Pull request merged**